serverless/src/adminGetRetrievals/handler.js

```diff
@@ -44,13 +45,26 @@ const adminGetRetrievals = async (event, context) => {
       )
       .select(dbConnection.raw('count(*) OVER() as total'))
       .join('users', { 'retrievals.user_id': 'users.id' })
+
+    if (userId) {
+      query = query.whereRaw('LOWER(users.urs_id) = LOWER(?)', [userId])
+    }
+
+    const retrievalResponse = await query
       .orderBy(...sortKeyMap[sortKey])
       .limit(pageSize)
       .offset((pageNum - 1) * pageSize)
 
-    const [firstResponseRow] = retrievalResponse
+    if (userId && retrievalResponse.length === 0) {
+      return {
+        isBase64Encoded: false,
+        statusCode: 404,
+        headers: defaultResponseHeaders,
+        body: JSON.stringify({ errors: [`No retrievals found for user: ${userId}`] })
```

Comment:
I don't think 404 is the way to respond to an empty result. What happens on the front end when a 404 is returned? I'm guessing we get our red error banner, which is really meant for something going wrong. 

When we search for collections in CMR and there are no results we (and CMR) don't return/show an error, just show that there are no results